### PR TITLE
feat/HNG-17-success-page-for-magic-link

### PR DESCRIPTION
**Pull request by oyin-systems**
### What does this PR do?
The changes in this pull request aim to implement the successful login with magic link page as shown in the design.

### Link to Design File
[Figma File](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=1-2&m=dev)

### Screenshot of my implementation
My implementation

### CODE REVIEW

**Comment by chumex412** [comment]
Please resolve this PR's conflicts

**Comment by oyin-systems** [comment]
> Please resolve this PR's conflicts

I just did that, please check again.

File: app/routes/magic-success.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <img
+          src={success}
+          alt="success"
+          className="mb-4 md:mb-8 w-[70px] md:w-[120px] h-[70px] md:h-[120px]"
+        />
+        <p className="font-normal text-base md:text-xl leading-6 text-[#0A0A0A] text-center mb-6 w-[320px] md:w-fit">
+          We have sent an email to talk2johnsnow@gmail.com. It contains
+          instructions on how to get started.
+        </p>
+        <button className="border border-[#CBD5E1] md:border-[#F97316] bg-white text-[#F97316] text-base font-medium leading-5 text-center h-10 md:h-16 md:w-full w-1/2 md:py-2 px-4 rounded-[6px]">
```
Comment (chumex412) [inline app/routes/magic-success.tsx:24]: Please go to the tailwind config and find the updated colors equivalent for all the colors you're using, e.g #0A0A0A

**Review by chumex412 — CHANGES_REQUESTED** [review]

**Comment by oyin-systems** [comment]
@chumex412 I've done the requested changes.

**Pull request closed without merging**